apps/web/src/pages/Pools/EditPoolPage/editPoolOperations.graphql

```diff
@@ -86,6 +86,13 @@ query getEditPoolData($poolId: UUID!) {
     stream
     processNumber
     publishingGroup
+    screeningQuestions {
+      id
+      question {
+        en
+        fr
+      }
+    }
```

Comment:
I feel sortOrder should be used to render the questions in the appropriate order on all pages, currently the order on a view/edit page does not exactly match the sortOrder. 

https://recordit.co/T5H31HvzeC

Reply:
I added this to the schema to default to sort by `sort_order`. I think that makes sense since I am having a hard time thinking of a scenario when we don't want to sort by that and if we do, we can always override it 🤷‍♀️ 

[default screening questions to order by sort_order](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6303/commits/d465a4ca79a6dd28271d2053cc419e85e9df90af)